cmd/gen.go

```diff
@@ -15,6 +15,10 @@ func GenCommand() cli.Command {
 				Name: "binary",
 				Usage: "",
 			},
+			cli.StringFlag{
+				Name: "binary-path",
+				Usage: "the directory to put binaries. works only when --binary is specified",
```

Comment:
@mumoshu 
https://github.com/k-kinzal/aliases/blob/master/cmd/gen.go#L37
`aliases` always export files for resolve of dependencies.
Therefore, we need to think about the behavior of `aliases gen --binary-path /path/to`.